lib/main.sh

```diff
@@ -39,22 +41,14 @@ backtitle="Armbian building script, http://www.armbian.com | Author: Igor Pecovn
 [[ -z $FORCE_CHECKOUT ]] && FORCE_CHECKOUT=yes
 
 # Load libraries
-# shellcheck source=debootstrap-ng.sh
-source "${SRC}"/lib/debootstrap-ng.sh 			# System specific install
-# shellcheck source=image-helpers.sh
-source "${SRC}"/lib/image-helpers.sh			# helpers for OS image building
-# shellcheck source=distributions.sh
-source "${SRC}"/lib/distributions.sh 			# System specific install
-# shellcheck source=desktop.sh
-source "${SRC}"/lib/desktop.sh 					# Desktop specific install
-# shellcheck source=compilation.sh
-source "${SRC}"/lib/compilation.sh 				# Patching and compilation of kernel, uboot, ATF
-# shellcheck source=makeboarddeb.sh
-source "${SRC}"/lib/makeboarddeb.sh 			# Create board support package
-# shellcheck source=general.sh
-source "${SRC}"/lib/general.sh				# General functions
-# shellcheck source=chroot-buildpackages.sh
-source "${SRC}"/lib/chroot-buildpackages.sh			# Building packages in chroot
+source "${SRC}"/lib/debootstrap-ng.sh 						# system specific install
```

Comment:
Let me know if you don't like the shellcheck hints there, I've got another round of shellcheck fixes so I can omit those.

Reply:
I do. Its just not my routine yet :)

Reply:
Why is this comment here needed?

Reply:
`shellcheck -x` will recognize it and then source the file referred in there